ui5/browser/controller/Browser.controller.js

```diff
@@ -214,12 +229,42 @@ sap.ui.define(['sap/ui/core/Component',
          var oReader = new FileReader();
          oReader.onload = function() {
             oModel.setProperty("/code", oReader.result);
-         }
+         };
          var file = oEvent.getParameter("files")[0];
          if (this.setFileNameType(file.name))
             oReader.readAsText(file);
       },
 
+     _getSettingsMenu: async function () {
+       if (!this._oSettingsMenu) {
+         let myThis = this;
+         await sap.ui.core.Fragment.load({name: "rootui5.browser.view.settingsmenu"}).then(function (oSettingsMenu) {
+           let oModel = new sap.ui.model.json.JSONModel("rootui5sys/browser/json/drawingOptions.json");
```

Comment:
Maybe we can avoid creation of extra JSON file and just put JSON into controller?
like:
```
new JSONModel({"TH1": [
		{
			"name": "hist"
		},
		...
})
```

Reply:
Also you should avoid use of fully-qualified class names like `sap.ui.model.json.JSONModel`.
They should be obtained via `sap.ui.define` (actually JSONModel is there)

Reply:
I mean i can, but the JSON is actually 200lines long, and only have TH1, TH2 and TProfile, if we add more, i will grow quite a lot, that is why i made it into a separate file.
But sure i can do it if you want